app/models/subscription.rb

```diff
@@ -26,6 +26,8 @@ def self.find_and_verify_by_token(token)
   end
 
   def search_criteria_to_h
```

Comment:
Once all the search criterias have been converted, can we lose this method?

Reply:
Yes, this is just the first step